unity/CAVE/Assets/W3D_Translator/CLI.cs

```diff
@@ -241,25 +251,76 @@ private static void BuildWalls(Story xml, Transform storyT)
                 // Set xml for button
                 button.targetGraphic = contentGO.GetComponent<Graphic>(); // Text, Image, etc.
                 button.colors = link.SetColors(button.colors, xml.ColorString);
-                if (!link.RemainEnabled)
-                { // TODO: Disable button after click
-                }
-
-                // TODO (83): Add button actions
-                foreach (LinkActions action in link.Actions)
-                {
-                }
             }
             else
             {
                 contentGO.SetActive(xml.Visible);
                 xml.Placement.SetTransform(contentGO.transform, xml.GetScale(), story.transform);
             }
-            gameObjects.Add(contentGO.name, contentGO);
+            gameObjects.Add(contentGO.name, (contentGO, xml));
         }
         return gameObjects;
     }
 
+    private static void SetLinkActions(ObjDictionary gameObjects, GameObject story)
+    {
+        ActionMethods methods = story.GetComponent<ActionMethods>();
+
+        foreach (KeyValuePair<string, (GameObject, W3D.Object)> pair in
+            gameObjects.Where(pair => pair.Value.Item2.LinkRoot is not null)
+        )
+        {
+            (GameObject go, W3D.Object obj) = pair.Value;
```

Comment:
Can you do `var (go, obj) = pair.Value` here? Or do you need to specify types? In general you're specifying the type a lot when you can have C# do that work for you

Reply:
I can but get a linter warning to specify the type when I do that. Do you think it's safe to disable the warning? I don't really mind all of the type definitions

Reply:
booo, I don't think these types are helpful, and the compiler can figure them out. I'd personally remove that rule, but it's up to you! If you like type definitions you can keep them

Reply:
Was pretty easy to disable in the `editorconfig`!